pkg/securitypolicy/framework.rego

```diff
@@ -1049,3 +1049,59 @@ errors[fragment_framework_svn_error] {
     semver.compare(data[input.namespace].framework_svn, svn) > 0
     fragment_framework_svn_error := concat(" ", ["fragment framework_svn is ahead of the current svn:", data[input.namespace].framework_svn, ">", svn])
 }
+
+errors["containers only distinguishable by allow_stdio_access"] {
+    input.rule == "create_container"
+
+    not container_started
+    possible_containers := [container |
```

Comment:
Yes but this was intentional as when we make this a better error message (coming in the not so distant future) this might no longer share all the logic. So I intentionally decided to not do that now given additional changes coming soon.

If you want it refactored now knowing that it might be undone when this is addressed further in March, I can do it. If come the changes in March, these still are rather similar, I was going to do the refactor to remove duplication then.

Reply:
Got it.